scss/_alert.scss

```diff
@@ -8,17 +8,27 @@
   --#{$prefix}alert-padding-x: #{$alert-padding-x};
   --#{$prefix}alert-padding-y: #{$alert-padding-y};
   --#{$prefix}alert-margin-bottom: #{$alert-margin-bottom};
+  @include rfs($font-size-base, --#{$prefix}alert-font-size); // Boosted mod
+  --#{$prefix}alert-line-height: #{$line-height-base}; // Boosted mod
   --#{$prefix}alert-color: #{$alert-color}; // Boosted mod
   --#{$prefix}alert-border-color: transparent;
   --#{$prefix}alert-border: #{$alert-border-width} solid var(--#{$prefix}alert-border-color);
   --#{$prefix}alert-border-radius: #{$alert-border-radius};
+  --#{$prefix}alert-logo-size: #{$alert-logo-size}; // Boosted mod
+  --#{$prefix}alert-icon-size: #{$alert-icon-size}; // Boosted mod
+  --#{$prefix}alert-icon-margin-y: #{$alert-icon-margin-y}; // Boosted mod
+  --#{$prefix}alert-link-font-weight: #{$alert-link-font-weight}; // Boosted mod
+  --#{$prefix}alert-heading-font-weight: #{$alert-heading-font-weight}; // Boosted mod
+  --#{$prefix}alert-dismissible-padding-right: #{$alert-dismissible-padding-r}; // Boosted mod
```

Comment:
Since the button is positioned with absolute values, it seems that `$alert-dismissible-padding-r` becomes useless, and so is this CSS variable.
If we confirm that, I will create a PR for Bootstrap since it is probably the same over there.

Reply:
I think it's to avoid long text to be under the close button.